javascript/spec/jasmine-web-spec.js

```diff
@@ -0,0 +1,67 @@
+import 'babel-polyfill'
+import 'colors';
+import wd from 'wd';
+
+const username = process.env.KOBITON_USERNAME
+const apiKey = process.env.KOBITON_API_KEY
+const deviceName = process.env.KOBITON_DEVICE_NAME || '*'
+
+let driver;
+const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api.kobiton.com',
+  auth: `${username}:${apiKey}`
+};
+const desiredCapabilities = {
```

Comment:
It just runs on Android devices

Reply:
files renamed, currently will support for android device (web/app), we'll update for iOS later.